packages/openneuro-app/src/scripts/front-page/front-page.jsx

```diff
@@ -25,39 +26,44 @@ class FrontPage extends React.Component {
 
   render() {
     return (
-      <span className="front-page is-front">
-        <div className="intro">
-          <div className="container">
-            <div className="intro-inner fade-in clearfix">
-              <div className="clearfix welcome-block">
-                {this._logoLayers()}
-                {frontPage.titlePanel.logoText ? this._logoText() : null}
-                <h1>{frontPage.pageDescription}</h1>
-                <LoggedOut>
-                  <div className="sign-in-block fade-in">
-                    <AuthenticationButtons />
+      <>
+        <Helmet>
+          <title>Home - {frontPage.pageTitle}</title>
```

Comment:
I think we should replace "Home" with the tagline (or a concise variation of it).